lib/utils/config/update.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const path = require('path');
+const getServerlessDir = require('./getServerlessDir');
+const writeFileSync = require('../fs/writeFileSync');
+
+function updateConfig(config) {
```

Comment:
Could we move this code to the `Utils` class in `lib/classes/Utils.js` since that's the central place for code like this.